Src/Witsml/Data/WitsmlMessageObject.cs

```diff
@@ -0,0 +1,21 @@
+using System.Xml.Serialization;
+
+namespace Witsml.Data
+{
+    [XmlRoot]
+    public class WitsmlMessageObject
+    {
+        [XmlAttribute("uidWell")] public string UidWell { get; set; }
```

Comment:
Might be a personal preference, but I find it easier to read when the attributes are on the line above instead of inline. The overall code base have them separated, so this style should be enforced here as well